translator/translate/otel/pipeline/host/translator.go

```diff
@@ -59,15 +62,27 @@ func (t translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators,
 	if conf == nil || t.receivers.Len() == 0 {
 		return nil, fmt.Errorf("no receivers configured in pipeline %s", t.name)
 	}
+	var entityProcessor common.Translator[component.Config]
+	if strings.HasPrefix(t.name, common.PipelineNameHostOtlpMetrics) {
+		entityProcessor = nil
+	} else if strings.HasPrefix(t.name, common.PipelineNameHostCustomMetrics) {
+		entityProcessor = awsentity.NewTranslatorWithEntityType(awsentity.Service)
+	} else if strings.HasPrefix(t.name, common.PipelineNameHost) || strings.HasPrefix(t.name, common.PipelineNameHostDeltaMetrics) {
+		entityProcessor = awsentity.NewTranslatorWithEntityType(awsentity.Resource)
+	}
```

Comment:
nit: Can move this into the block below. No point creating the translator if the destination isn't CloudWatch.

Reply:
Non-blocking